R/prepare_matches_mapper.R

```diff
@@ -12,10 +12,11 @@ prepare_matches_mapper <- function(mapper, activities) {
   mapper |>
     left_join(activities, by = "activity_uuid_product_uuid") |>
     summarise(across(everything(), ~ paste0(na.omit(unique(.)), collapse = "; ")), .by = c("country", "main_activity", "clustered")) |>
-    ungroup() |>
+    mutate(across(everything(), ~ifelse(. == "", NA_character_, .))) |>
     distinct() |>
+    mutate(multi_match = as.logical(.data$multi_match)) |>
```

Comment:
As I explain in another comment, if we assert that `multi_match` is logical, then we don't need this line. In that case what we do need is a test to ensure that passing `multi_match` of type "character" throws the expected error.

Reply:
@maurolepore Added test!